src/main/java/com/urswolfer/intellij/plugin/gerrit/push/GerritPushExtensionPanel.java

```diff
@@ -200,6 +202,11 @@ private void createLayout() {
                 "Include a hashtag associated with all of the changes in the same group.",
                 9);
 
+        patchsetDescriptionTextField = addTextField(
+                "Patch Set Description (Gerrit 3.4+):",
+                "A description of the patch set to be created. Intended to help guide reviewers as a change evolves. The description cannot be changed after the change is pushed.",
+                10);
```

Comment:
I have missed this number overlap with the next one - it causes an UI glitch. Would you like to create a PR?

See also: https://github.com/uwolfer/gerrit-intellij-plugin/issues/502

Reply:
I have fixed it.